Tools/build/check_extension_modules.py

```
@@ -281,6 +289,38 @@ def list_module_names(self, *, all: bool = False) -> set[str]:
             names.update(WINDOWS_MODULES)
         return names
 
+    def generate_stdlib_info(self, config_path: str | None = None) -> None:
+        config_messages = {}
+        if config_path:
+            try:
+                with open(config_path, encoding='utf-8') as f:
+                    config_messages = json.load(f)
+            except (FileNotFoundError, json.JSONDecodeError) as e:
+                logger.error("Failed to load distributor config %s: %s", config_path, e)
```

Could you also raise the exception? If the user asked for something explicitly we should fail if we can't do it.